web/src/components/product/ProductSelectionPage.jsx

```diff
@@ -46,34 +47,57 @@ function ProductSelectionPage() {
     }
   };
 
-  const Item = ({ children }) => {
+  const Item = ({ children }) => (
+    <GridItem sm={10} smOffset={1} lg={8} lgOffset={2} xl={6} xlOffset={3}>
```

Comment:
I expect this padding in CSS, but sadly I am not css expert. Lets hope @dgdavid gets back soon